### DISCUSSION TITLE
How can I change the theme color dynamically in v3.0.0-alpha.12

### DISCUSSION THREAD

**Discussion by dimacm** [vuetifyjs/vuetify#14476] [Q&A]
I want to dynamically change the theme colors.
In previous versions it was possible to do this.
But now, the parameter is changing, but it does not affect the color display in ui.

`this.$vuetify.theme.themes.green.colors.background = '#ff0000'` - no longer changes color dynamically

Help please.

**Comment by radimhornicek** [comment]
I can confirm that. I had similar problem. In Nuxt 3 I needed to initiate primary color which depends on user's data (fetched by API). There is a workaround for this specific case - in my plugins folder I have first two files like 1.init.js and 2.vuetify.js, where in init.js plugin I call  API for data (e.g. - I use PINIA action with useFetch for this). Nuxt's definePlugins func must be async/await and then in 2.vuetify.js just import my state from pinia store, take user's data and set options for vuetify when vuetify is initalized by createVuetify func.

It's not exactly what you've asked, but maybe this could possibly help anyone instead. But dynamic change would make it easier for me.
